integration_test/ops_agent_test.go

```diff
@@ -28,13 +28,16 @@ The following variables are optional:
 
 REPO_SUFFIX: If provided, what package repo suffix to install the ops agent from.
 ARTIFACT_REGISTRY_REGION: If provided, signals to the install scripts that the
-    above REPO_SUFFIX is an artifact registry repo and specifies what region it
-	is in. If not provided, that means that the packages are accessed through
-	packages.cloud.google.com instead, which may point to Cloud Rapture or
-	Artifact Registry under the hood.
+
```

Comment:
`gofmt` did this automatically. I think it's a `godoc` convention. Maybe we should reformat this comment if we don't want it to look like this?